sdk/devcenter/ci.yml

```diff
@@ -0,0 +1,31 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
we have updated the ci.yml to ci.mgmt.yml, the also change the setting, you can refer https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/avs/ci.mgmt.yml

Reply:
Renamed and updated the file